lib/pushNotifications.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:io';
+
+import 'package:firebase_messaging/firebase_messaging.dart';
+
+class PushNotificationsManager {
+  PushNotificationsManager._();
+
+  factory PushNotificationsManager() => _instance;
+
+  static final PushNotificationsManager _instance =
+      PushNotificationsManager._();
+
+  final FirebaseMessaging _firebaseMessaging = FirebaseMessaging();
+  bool _initialized = false;
+
+  Future<void> init() async {
+    if (!_initialized) {
+      // For iOS request permission first.
```

Comment:
Can you partner up with someone who has iOS and Mac to implement notifications for iOS?

Reply:
You have me, @upDJ, & @davidsquines to work with

Reply:
rename method to something more readible like `initializePushNotificationsManager()`

Reply:
bad news yall, in order for us to send notifications to ios devices we need an Apple Developer account which costs $99 dolllazzz (ayyyyy :zany_face: ) and I don't think it's worth it to purchase now as we're about 68% done. 

So until we have our application working on android that's when we'll focus on deploying onto the apple store and integration the notifications feature for ios devices.